chat-widget/src/components/livechatwidget/common/startChat.ts

```diff
@@ -67,11 +67,19 @@ const setPreChatAndInitiateChat = async (chatSDK: any, dispatch: Dispatch<ILiveC
     const parseToJson = false;
     const preChatSurveyResponse: string = await chatSDK.getPreChatSurvey(parseToJson);
     const showPrechat = isProactiveChat ? preChatSurveyResponse && proactiveChatEnablePrechatState : (preChatSurveyResponse && !props?.controlProps?.hidePreChatSurveyPane);
+    const isOutOfOperatingHours = state?.domainStates.liveChatConfig?.LiveWSAndLiveChatEngJoin?.OutOfOperatingHours === "True";
 
     if (showPrechat) {
-        dispatch({ type: LiveChatWidgetActionType.SET_PRE_CHAT_SURVEY_RESPONSE, payload: preChatSurveyResponse });
-        dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.Prechat });
-        return;
+        if (isOutOfOperatingHours === true) {
+            state?.appStates.isMinimized && dispatch({ type: LiveChatWidgetActionType.SET_MINIMIZED, payload: false });
+            dispatch({ type: LiveChatWidgetActionType.SET_CONVERSATION_STATE, payload: ConversationState.Loading });
```

Comment:
Two `SET_CONVERSATION_STATE` events were dispatched. Are both needed?

Reply:
The OutOfOffice state alone is not rendering the OOH pane correctly. Seems like the state needs to be handled separately.

![image](https://github.com/microsoft/omnichannel-chat-widget/assets/49963796/d2d2d71e-4a63-4356-88d1-2315fd376f39)

Reply:
I also don't quite understand this part. Line 75's effect will be instantly overwritten in this case.

@akhil In the example you gave, we needed to render the loading pane for a while first before rendering other panes